src/NUnitFramework/testdata/TestFixtureSourceData.cs

```diff
@@ -72,10 +72,7 @@ public StaticProperty_SameClass(string arg, string expected) : base(arg, expecte
         {
         }
 
-        public static object[] StaticProperty
-        {
-            get { return new object[] { new object[] { "StaticPropertyInClass" } }; }
-        }
+        public static object[] StaticProperty => new object[] { new object[] { "StaticPropertyInClass" } };
```

Comment:
Revert this unrelated change as it is not related to .net 9.0 change

Reply:
Reverted it. Sorry for committing in the first place, happened on accident by my lsp